dev/docs/windows-source-installation.md

```diff
@@ -0,0 +1,81 @@
+# Developing Apache SystemDS on Windows Platform
+
+These instructions will help you build Apache SystemDS from source code, which is the basis for engine and algorithms development.
+The following conventions will be used to refer to directories on your machine:
+
+* `<USER_HOME>` is your home directory.
+* `<MAVEN_HOME>` is the root directory for the Apache Maven source code.  
+* `<SYSTEMDS_HOME>` is the root directory for the SystemDS source code.
+* `<SPARK_HOME>` is the root directory for the Apache Spark source code.
+* `<SCALA_HOME>` is the root directory for the Scala.
+Ex. `C:\Program Files (x86)\scala`
+* `<SBT_HOME>` is the root directory for the Scala SBT.
```

Comment:
Scala And SBT is not needed anymore, since the entire scala part is not in use anymore/ removed for simplicity

Reply:
Ok.